src/main/java/org/codeforamerica/shiba/application/parsers/ApplicationDataParser.java

```diff
@@ -11,11 +14,11 @@ public ApplicationDataParser(ParsingConfiguration parsingConfiguration) {
 
     public abstract T parse(ApplicationData applicationData);
 
-    protected static double getDouble(ApplicationData applicationData, PageInputCoordinates pageInputCoordinates) {
+    protected static Money getMoney(ApplicationData applicationData, PageInputCoordinates pageInputCoordinates) {
         try {
-            return Double.parseDouble(applicationData.getValue(pageInputCoordinates).replace(",",""));
+            return new Money(new BigDecimal(applicationData.getValue(pageInputCoordinates).replace(",","")));
```

Comment:
Can `parseWithCommasRemoved` be reused here?

Reply:
Removed this in new changes